composer.json

```diff
@@ -27,6 +27,7 @@
         "doctrine/dbal": "^2.8",
         "neos/party": "*",
         "neos/setup": "*",
+        "neos/neos-setup": "*",
```

Comment:
hey,

IMHO this file is autogenerated; why did you add neos-setup here?

all the best,
Sebastian